worlds/ror2/docs/en_Risk of Rain 2.md

```diff
@@ -31,13 +57,34 @@ in-game item of that tier will appear in the Risk of Rain player's inventory. If
 the player already has an equipment item equipped then the _item that was equipped_ will be dropped on the ground and _
 the new equipment_ will take it's place. (If you want the old one back, pick it up.)
 
+### How many items are there?
+
+Since a Risk of Rain 2 run can go on indefinitely, you have to configure how many collectible items (also known as
+"checks") the game has for purposes of Archipelago when you set up a multiworld. You can configure anywhere from **10
+to 500** items. The number of items will be randomized between all players, so you may want to adjust the number and
```

Comment:
Decision has finally been made to reduce this number to 250 which I just finished doing here https://github.com/ArchipelagoMW/Archipelago/pull/776